tests/functional/aws-node-sdk/test/multipleBackend/objectCopy.js

```diff
@@ -86,6 +85,10 @@ destBucket, destLoc, awsKey, mdDirective, isEmptyObj, callback) {
             assert.deepStrictEqual(sourceRes.Metadata['test-header'],
                 destRes.Metadata['test-header']);
         }
+        if (destLoc === awsLocation) {
+            // if copying to AWS  => retain metadata even though directive: COPY
```

Comment:
do you mean `even though directive: REPLACE`?
do you mean we retain all the metadata even if directive is `REPLACE` or just the location metaheader?